.github/notebooks-config.json

```diff
@@ -34,6 +34,11 @@
       "name": "nom",
       "path": "NOM_visualizations/python/nom_python.ipynb",
       "display_name": "Natural Organic Matter Visualizations"
+    },
+    {
+      "name": "metagmetat",
+      "path": "metaG_metaT_integration/python/metagmetat_r.ipynb",
```

Comment:
```suggestion
      "path": "metaG_metaT_integration/python/metagmetat_python.ipynb",
```

Reply:
Ah... Sorry, I miss typing that notebook name.